tests/modules/local/maptotsv/main.nf.test

```diff
@@ -0,0 +1,289 @@
+nextflow_process {
+
+    name "Test Process MAP_TO_TSV"
+    script "modules/local/map_to_tsv.nf"
+    process "MAP_TO_TSV"
+
+    test("Single row, basic input") {
+
+        when {
+            params {
+                outdir = "tests/results"
+            }
+            process {
+                """
+                input[0] = new Tuple("sample", "outbreak", "organism", "subtype", "country", "serovar", "age", "date", "source", "special")
+                input[1] = new Tuple(["S1", "1", "Escherichia coli", "EHEC/STEC", "Canada", "O157:H7", "21", "2024/05/30", "beef", "true"])
+                """
+            }
+        }
+
+        then {
+            assert process.success
+
+            with(process.out) {
+                // check outputs
+                assert path(tsv_path[0]).exists()
+                assert nonempty_column_headers == [["sample", "outbreak", "organism", "subtype", "country", "serovar", "age", "date", "source", "special"]]
+
+                // parse tsv file
+                def tsv_text = path(tsv_path[0]).readLines()
+                assert tsv_text.contains("S1\t1\tEscherichia coli\tEHEC/STEC\tCanada\tO157:H7\t21\t2024/05/30\tbeef\ttrue")
+            }
+        }
+
+    }
+
+    test("Single row, empty columns, head") {
+
+        when {
+            params {
+                outdir = "tests/results"
+            }
+            process {
+                """
+                input[0] = new Tuple("sample", "outbreak", "organism", "subtype", "country", "serovar", "age", "date", "source", "special")
+                input[1] = new Tuple(["S1", "1", "", "", "", "", "21", "2024/05/30", "beef", "true"])
+                """
+            }
+        }
+
+        then {
+            assert process.success
+
+            with(process.out) {
+                // check outputs
+                assert path(tsv_path[0]).exists()
+                assert nonempty_column_headers == [["sample", "outbreak", "age", "date", "source", "special"]]
+
+                // parse tsv file
+                def tsv_text = path(tsv_path[0]).readLines()
+                assert tsv_text.contains("S1\t1\t21\t2024/05/30\tbeef\ttrue")
+            }
+        }
+
+    }
+
+    test("Single row, empty columns, tail") {
+
+        when {
+            params {
+                outdir = "tests/results"
+            }
+            process {
+                """
+                input[0] = new Tuple("sample", "outbreak", "organism", "subtype", "country", "serovar", "age", "date", "source", "special")
+                input[1] = new Tuple(["S1", "1", "Escherichia coli", "EHEC/STEC", "Canada", "O157:H7", "", "", "", ""])
+                """
+            }
+        }
+
+        then {
+            assert process.success
```

Comment:
Could you add additional asserts here on the contents of the output file like the other test cases.

Reply:
Yeah, this is weird. I wonder why I left it empty. Haha.

Reply:
Added in 0ebd7271c224d509091af468c63bf5313067bcb5